applications/FemToDemApplication/python_scripts/CouplingFemDem.py

```diff
@@ -946,6 +946,7 @@ def InitializeIntegrationPointsVariables(self):
             elem.SetValue(KratosFemDem.SMOOTHING, 0)
```

Comment:
This is slow, use VariableUtils

Reply:
Not this

Reply:
Should I use `KratosMultiphysics.VariableUtils().SetNonHistoricalVariable` ?

Reply:
Yep